includes/class-wc-payments-account.php

```diff
@@ -1264,7 +1265,7 @@ private function get_actioned_notes(): array {
 			return $where_clause . " AND name like 'wcpay-promo-%'";
 		};
 
-		$note_class = WC_Payment_Woo_Compat_Utils::get_note_class();
+		$note_class = Note::class;
```

Comment:
Similar to notest classes here.

Reply:
Thanks, done here 
[9434390](https://github.com/Automattic/woocommerce-payments/pull/5331/commits/9434390dc3696f83f9b89ab8d94fdac0de98b8d3)